src/window.rs

```diff
@@ -114,6 +114,12 @@ pub struct WindowAttributes {
     /// The default is `None`.
     pub max_inner_size: Option<LogicalSize>,
 
+    /// The desired position of the window. If this is `None`,
+    /// some platform-specific position will be chosen.
+    ///
+    /// The default is `None`.
```

Comment:
We mention in `with_outer_position` that this doesn't work on Wayland. We should mention that here, too.